src/blockdata/transaction.rs

```diff
@@ -426,6 +426,39 @@ impl Transaction {
         self.get_scaled_size(1)
     }
 
+    /// Gets the "vsize" of this transaction. Will be `ceil(weight / 4.0)`.
+    #[inline]
+    pub fn get_vsize(&self) -> usize {
+        let weight = self.get_weight();
+        (weight + WITNESS_SCALE_FACTOR - 1) / WITNESS_SCALE_FACTOR
+    }
+
+    /// Gets the size of this transaction excluding the witness data.
+    pub fn get_strippedsize(&self) -> usize {
```

Comment:
Can you add a test with a few sample transactions where you call `get_strippedsize` on the original transaction, then remove the witness, then check that the new `vsize`/`weight`/`size` methods all return the same as the original `strippedsize` (mod multipliers).

Reply:
I don't think it is needed because `test_nonsegwit_transaction()` covers this, but I added this in https://github.com/rust-bitcoin/rust-bitcoin/pull/626/commits/cdf7be47654041ac375c80332bad0affb69c889a.